embedchain/config/apps/CustomAppConfig.py

```diff
@@ -21,19 +21,20 @@ def __init__(
         embedding_fn: EmbeddingFunctions = None,
         embedding_fn_model=None,
         db=None,
+        collection_name=None,
         host=None,
         port=None,
         id=None,
         provider: Providers = None,
         model=None,
         open_source_app_config=None,
-    ):
```

Comment:
please close the bracket, sorry must've been a merge mistake I did.

Reply:
Thanks for fixing the merge conflicts! And fixed.